apps/api/src/app/db/content/content.store.ts

```diff
@@ -108,12 +107,11 @@ export class ContentStore {
             return doc;
         } else {
             const authorInfo = doc.author as any;
-            if (authorInfo._id === user.sub) {
-                return doc;
-            } else {
+            if (authorInfo._id !== user.sub) {
                 await this.incrementViewCount(contentId);
-                return doc;
             }
+            await this.history.addOrUpdateHistory(user, contentId);
+            return doc;
         }
```

Comment:
So I recognize that a lot of this is an artifact from not having adjusted from the old design, where Stores had a lot of responsibilities--but the ContentStore should 100% not be touching the HistoryStore--there should be a ContentService (which I believe already exists?) in front of this that fetches from the ContentStore, and updates the user's History in the HistoryStore.

I don't think this will ever get fixed unless we rip of the band-aid and start doing things properly, so I'm going to block this PR on this.